finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Sentinels.scala

```diff
@@ -0,0 +1,181 @@
+package com.twitter.finagle.redis.protocol.commands
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+import com.twitter.finagle.redis.ClientError
+
+import java.net.InetSocketAddress
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+
+case class SentinelMaster(name: ChannelBuffer)
+  extends Sentinel(SentinelMaster.channelBuffer, Seq(name))
+
+object SentinelMaster extends SentinelHelper {
+  val command = "MASTER"
+  def apply(args: Seq[Array[Byte]]): SentinelMaster = {
+    val name = trimList(args, 2, "SENTINEL MASTER")(0)
+    new SentinelMaster(ChannelBuffers.wrappedBuffer(name))
+  }
+}
+
+case class SentinelMasters() extends Sentinel(SentinelMasters.channelBuffer, Nil)
```

Comment:
that seems even sillier than case class with no constructor args.  how about you leave it this way and we'll change it when we change the other case classes with no constructor args.